public/app/plugins/datasource/graphite/specs/graphite_query.test.ts

```diff
@@ -74,4 +74,25 @@ describe('Graphite query model', () => {
       expect(ctx.queryModel.functions[1].params[0]).toBe('$limit');
     });
   });
+
+  describe('skips "select metric" when target is rendered', () => {
```

Comment:
can we find a better description for this test? it's not very clear to me who is skipping what, and what does "skips" mean in this context.

Reply:
better dfd1e571a483ff6f8d201e7065b0e80b66936960?

Reply:
lovely <3